data/growbydata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "GrowByData Services Pvt. Ltd.",
+  "establishedAt": 1646072100,
```

Comment:
Everything looks good, but can I get the source of this company establishment date?

Reply:
@thenishantsapkota Got to know though this article : https://omgnepal.com/growbydata-data-revolution-in-nepal/